extras/create_snapshots_reports_scorecard.py

```diff
@@ -40,8 +40,8 @@
 
 LOGGING_LEVEL = logging.INFO
 LOG_FILE = "snapshots_reports_scorecard_automation.log"
-REPORT_THREADS = 40
-SNAPSHOT_THREADS = 40
+REPORT_THREADS = 70
+SNAPSHOT_THREADS = 70
```

Comment:
I think we can bump up the thread count a little bit more still.  There are 96 CPUs on the DB instance, so we might be able to go a bit higher than 80 for `SNAPSHOT_THREADS`, but we can be conservative here since we are now generating all of our snapshots in ~20 minutes.

```suggestion
REPORT_THREADS = 72  # Match the number of available CPUs on the reporter instance
SNAPSHOT_THREADS = 80
```

Reply:
Please see commit ebdd10e.